README.md

```diff
@@ -175,6 +175,12 @@ The option `country_display_type` can be specified to change the country dropdow
 | `display_country_full` (default)  |  United Kingdom (+44)  |
 | `display_country_short`           |  GB +44                |
 
+And with the option `country_display_emoji_flag` set to `true` (default is `false`) you can add the emoji flag of the country before the label :
```

Comment:
Why default false ? :)

Reply:
wdyt @odolbeau

Reply:
If set to true then people updating the bundle risk to have their design break by the added emoji. With false by default the update won't break anything and people keep full control over their design.

Reply:
IMO changing the default value to true may be done with a future major release and document as BC break. And even better with a deprecation notice triggered  before that ;)

Reply:
WDYT about setting it to true in the next major version?

Reply:
Sure, why not. :)

Reply:
Changelog updated